OpenOversight/migrations/versions/cd39b33b5360_constrain_officer_gender_options.py

```diff
@@ -11,19 +11,21 @@
 
 # revision identifiers, used by Alembic.
 revision = 'cd39b33b5360'
-down_revision = '562bd5f1bc1f'
+down_revision = '86eb228e4bc0'
```

Comment:
This is necessary to keep the migrations linear (exactly one "head"). Basing the new migration on the (now) most recent one